Makefile.js

```diff
@@ -76,9 +76,14 @@ const NODE = "node ", // intentional extra space
     MARKDOWNLINT_IGNORE_INSTANCE = ignore().add(fs.readFileSync(path.join(__dirname, ".markdownlintignore"), "utf-8")),
     MARKDOWN_FILES_ARRAY = MARKDOWNLINT_IGNORE_INSTANCE.filter(find("docs/").concat(ls(".")).filter(fileType("md"))),
     TEST_FILES = "\"tests/{bin,conf,lib,tools}/**/*.js\"",
-    PERF_ESLINTRC = path.join(PERF_TMP_DIR, "eslintrc.yml"),
+    PERF_ESLINTRC = path.join(PERF_TMP_DIR, "eslint.config.js"),
     PERF_MULTIFILES_TARGET_DIR = path.join(PERF_TMP_DIR, "eslint"),
-    PERF_MULTIFILES_TARGETS = `"${PERF_MULTIFILES_TARGET_DIR + path.sep}{lib,tests${path.sep}lib}${path.sep}**${path.sep}*.js"`,
+
+    /*
+     * glob arguments with Windows separator `\` don't work:
+     * https://github.com/eslint/eslint/issues/16259
+     */
+    PERF_MULTIFILES_TARGETS = `"${TEMP_DIR}/eslint/performance/eslint/{lib,tests/lib}/**/*.js"`,
```

Comment:
```suggestion
    PERF_MULTIFILES_TARGETS = `"${TEMP_DIR}eslint/performance/eslint/{lib,tests/lib}/**/*.js"`,
```

There is an extra `/` because `TEMP_DIR` already ends with a `/`.